hooks/useFuturesMarketClosed.ts

```diff
@@ -12,11 +11,11 @@ const useFuturesMarketClosed = (currencyKey: CurrencyKey | null) => {
 	const isFutureMarketSuspended =
 		futuresMarketSuspendedQuery.isSuccess && futuresMarketSuspendedQuery.data
 			? futuresMarketSuspendedQuery.data.isFuturesMarketClosed
-			: false;
+			: null;
```

Comment:
returning `null` here so we know when its a legitamate closed or open market (true/false) or the underlying query is not done yet (null).